tests/StorageApiTestCase.php

```diff
@@ -183,7 +183,7 @@ protected function initEmptyBucket($name, $stage, $description, Client $client =
             });
             foreach (array_reverse($tables) as $table) {
                 try {
-                    $client->dropTable($table['id']);
+                    $client->dropTable($table['id'], ['force' => true]);
```

Comment:
tady to je divno věc občas spadne test protože tady přitom dropu existuje alias, ale nepodařilo sa mě to zreplikovat.